src/main/java/io/strimzi/kafka/metrics/PrometheusMetricsReporterConfig.java

```diff
@@ -7,50 +7,79 @@
 import io.prometheus.client.exporter.HTTPServer;
 import org.apache.kafka.common.config.AbstractConfig;
 import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.net.BindException;
+import java.net.InetSocketAddress;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Optional;
+import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 /**
- * Configuration for the reporter implementations.
+ /**
```

Comment:
Do you have twice `/**`? Or is that some bad rendering in GitHub UI?

Reply:
Duplicate removed